### ISSUE TITLE
Parsing for Japanese is off seperates at punctuation

### ISSUE THREAD

**Issue by kaosine** [ll-in-anki/anki-LL#12]
Decided after remembering I saw this plugin on reddit to finally put it to use, and see that it actually runs into some  issues. Doesn't report any but it doesn't seperate at words, instead at punctuation like 、and 。which isn't correct. 

Article: https://www.bbc.com/japanese/features-and-analysis-51232723

(I would post a image of it, but apparently my theme no matter what I do messes with the green/white combo that this plugin tries to use)

**Comment by cofinley** [comment]
Yes, this was made initially from my perspective learning a language with spaces. Japanese/Chinese is obviously a different story but is a little difficult to manage. Morphman distinguishes between the two and is pretty involved last time I checked. But since it’s all open-source, it would be a good template for a change to this repo.
